tests/test_bytes_subclass_support.py

```diff
@@ -56,35 +64,38 @@ def test_handles_memoryviews(self) -> None:
         Ensure support for libraries that leverage memoryviews
         """
 
-        def base64url_to_memoryview(data: str) -> memoryview:
-            data_bytes = base64url_to_bytes(data)
-            return memoryview(data_bytes)
+        def as_memoryview(data: Union[str, bytes]) -> memoryview:
+            return memoryview(data.encode("ascii") if isinstance(data, str) else data)
```

Comment:
Similar to above, what prompted this refactor? I can't see it being obviously driven by changes in Pydantic v2, but maybe I'm missing something.